lib/extensions/src/lib/extensions.module.ts

```diff
@@ -15,10 +15,15 @@
  * limitations under the License.
  */
 
-import { NgModule } from '@angular/core';
 import { DynamicExtensionComponent } from './components/dynamic-component/dynamic.component';
 import { DynamicTabComponent } from './components/dynamic-tab/dynamic-tab.component';
 import { DynamicColumnComponent } from './components/dynamic-column/dynamic-column.component';
+import { NgModule, ModuleWithProviders, APP_INITIALIZER } from '@angular/core';
+import { AppExtensionService } from './services/app-extension.service';
```

Comment:
Renaming this service will break many stuff partners build for us, as they import and use it as ExtensionService, the App prefix was for application level and demo shell only.